### PR TITLE
fix: Deployment does not work from travis

### LINKED ISSUE #30: Deployment does not work from travis
### Describe the bug
As a developer I would like to deploy compiled code to `master` by merging `develop` branch into master so I could see freshly deployed version under  https://anitab-org.github.io

### To Reproduce
1. Merge develop branch into production
2. See travis did not push to master, only to gh-pages

### Expected behavior
I want to merge develop into production so I could see changes on deployed app. If possible ot also can be solved by github actions.

### PR DESCRIPTION
**Pull request by annabauza**
### Description

Deployment script was not working, this PR is fixing the issue. 
This PR include:
- Running building scripts on pre deploy stage
- define folder to upload to master
- limit deployment branch to master -  no longer gh-pages

Fixes #30

### Type of Change:
**Delete irrelevant options.**

- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,10 +2,17 @@ language: node_js
 node_js:
   - 8.15.0
 
+before_deploy:
+  - yarn
+  - yarn build
+
 deploy:
+  deploy:
```
Comment (sidvenu) [inline .travis.yml:10]: Why is `deploy:` being repeated in lines #9 and #10?

Comment (annabauza) [reply .travis.yml:10]: probably merge mistake 🤦‍♀️

**Comment by annabauza** [comment]
@sidvenu good catch thanks I have removed the duplication.

**Comment by annabauza** [comment]
@sidvenu refresh it is there...

**Pull request merged**